src/main/java/org/folio/roles/service/loadablerole/LoadableRoleCapabilityAssignmentHelper.java

```diff
@@ -129,4 +134,8 @@ private static Stream<Entry<UUID, List<LoadablePermissionEntity>>> groupPermissi
   private static Map<String, LoadablePermissionEntity> permissionsByName(List<LoadablePermissionEntity> perms) {
     return toStream(perms).collect(toMap(LoadablePermissionEntity::getPermissionName, identity()));
   }
+
+  private static Predicate<Capability> noneTechnical() {
```

Comment:
"noneTechnical"? Why not "notTechnical"?

Reply:
updated